h/templates/accounts/signup-successful.html.jinja2

```diff
@@ -0,0 +1,15 @@
+{% extends "h:templates/accounts/base.html.jinja2" %}
+
+{% block page_title %}{{ heading|default("Sign up for Hypothesis") }}{% endblock %}
+
+{% block form_message %}
+  {% if not form %}
+    {% if message %}<p>{{ message }}</p>{% endif %}
+
+    <p>Please check your email and open the link to activate your account. If you
+    don't receive your activation link within a few minutes, send an email to
+    <a href="mailto:support@hypothes.is">support@hypothes.is</a></p>
+  {% endif %}
+{% endblock %}
+
+{% set login_message = "Already activated your account?" %}
```

Comment:
Previously the sign-up page (with the form) and the account-registraction-successful page that you get taken to after submitting the sign-up form, shared the same template. They're really two completely different pages so this just resulted in a template of ifs and elses. I simplified it by splitting it into two separate, almost logic-less templates, in this commit https://github.com/hypothesis/h/pull/5964/commits/e0df6db8dc29e34c5ede2a8e0b7afa03165f4d30